video/out/opengl/context_wayland.c

```diff
@@ -144,6 +144,12 @@ static void egl_create_window(struct ra_ctx *ctx)
     }
 
     eglMakeCurrent(p->egl_display, p->egl_surface, p->egl_surface, p->egl_context);
+    // eglMakeCurrent may not configure the draw or read buffers if the context
+    // has been made current previously.
```

Comment:
Probably should make it clear that this is for NVIDIA.

Reply:
Updated.